0800-appendix-57793-vallee.Rmd

```diff
@@ -77,42 +116,71 @@ fpr::fpr_table_cv_summary_memo()
 
 ```
 
-
 ```{r eval=F}
 ##this is useful to get some comments for the report
 hab_site %>% filter(site == my_site & location == 'ds') %>% pull(comments)
 fpr::fpr_my_priority_info()
 
 ```
 
-## Stream Characteristics Downstream {-}
+## Stream Characteristics Downstream {.unnumbered}
 
-`r fpr_my_survey_dist(loc = 'ds')` to the confluence with the Bulkley River
+`r fpr_my_survey_dist(loc = 'ds')` to the confluence with the Bulkley
+River
```

Comment:
omg - you surveyed all the way to the Bulkley! Cool.  Was that like a 2.5km survey?  Revise